doc/source/conf.py

```diff
@@ -124,77 +126,14 @@
         ("PyAnsys Documentation", "https://docs.pyansys.com"),
         ("Shared Components", "https://shared.docs.pyansys.com"),
     ],
+    "switcher": {
+        "json_url": f"https://{cname}/versions.json",
```

Comment:
Does this mean the default is `https://https://ansys.github.io/openapi-common/versions.json`?

Reply:
Fixed, thanks